src/bout++.cxx

```diff
@@ -715,6 +715,13 @@ void addBuildFlagsToOptions(Options& options) {
   options["use_backtrace"].force(bout::build::use_backtrace);
   options["use_color"].force(bout::build::use_color);
   options["use_openmp"].force(bout::build::use_openmp);
+  options["openmp_threads"].force(
+#ifdef _OPENMP
+      omp_get_max_threads()
```

Comment:
It's a long time since I've worked with BOUT++ internals, but could the `ifdef` be replaced by a check of `options["use_openmp"]`?